src/ipoplib.py

```diff
@@ -17,8 +17,124 @@
 import time
 import urllib2
 import keyring
+import base64
+import traceback
 
 from threading import Timer
+from keyring.util.escape import escape as escape_for_ini
+from keyring.py27compat import configparser
+import keyring.backend
+from keyring.backends.file import Encrypted,BaseKeyring
+from keyring.util import properties
+
+class IPOPEncryptedKeyring(Encrypted, BaseKeyring):
```

Comment:
Right. I totally agree with you. we can probably use plaintext keyring. As we are using a fixed (publicly available)key to encrypt, it is as good as using a plaintext keyring.